src/components/molecules/ChatList/ChatMessage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from "react";
+
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import { WithId } from "utils/id";
+import { formatUtcSeconds } from "utils/time";
+import { getLinkFromText } from "utils/getLinkFromText";
+
+import { useUser } from "hooks/useUser";
+
+import { User } from "types/User";
+
+import {
+  PrivateChatMessage,
+  RestrictedChatMessage,
+} from "components/context/ChatContext";
+
+interface ChatMessageProps {
+  message: WithId<RestrictedChatMessage | PrivateChatMessage>;
+  usersById: Record<string, User>;
+  allowDelete: boolean;
+  onAvatarClick: (
+    message: WithId<RestrictedChatMessage | PrivateChatMessage>
+  ) => void;
+  onDeleteClick: (
+    message: WithId<RestrictedChatMessage | PrivateChatMessage>
+  ) => void;
+}
+
+export const ChatMessage: FC<ChatMessageProps> = ({
+  message,
+  usersById,
+  allowDelete,
+  onAvatarClick,
+  onDeleteClick,
+}) => {
+  const { user } = useUser();
+
+  const profileImageSize = 30;
+  const sender = { ...usersById[message.from], id: message.from };
+  const isMe = sender.id === user!.uid;
+  const profileImage = sender.anonMode
+    ? DEFAULT_PROFILE_IMAGE
+    : sender.pictureUrl;
+  const profileName = sender.anonMode ? DEFAULT_PARTY_NAME : sender.partyName;
+
+  return (
+    <>
+      {message.from in usersById && (
+        <>
```

Comment:
[NB] Shouldn't need the extra fragment wrapper here as there is only one element (`div`), so can just use it directly.

Reply:
This is stuff we can eventually catch and autofix with `eslint` and friends too.